### PR TITLE
Update R geoscience packages

### PR DESCRIPTION
**Pull request by choldgraf**
This is a re-opening of #101 , because the image build failed and so we reverted it from before.

We should:

- [x] Build the image locally, confirm that it builds, and push to the staging hub
- [ ] Confirm that it works w/ the instructor, by making sure that this notebook works: https://jupyter.utoronto.ca/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2Fhhwagner1%2FBIO311_Labs&urlpath=tree%2FBIO311_Labs%2F&branch=main
- [x] Merge this, and then merge to prod

mostly addresses https://github.com/2i2c-org/pilot-hubs/issues/697

### CODE REVIEW

File: deployments/utoronto/image/install.R [hunk 1]
```diff
@@ -9,78 +9,90 @@ install.packages("devtools")
...
   "scatterplot3d", "0.3-41",
-  "scholar", "0.1.7",
+  "scholar", "0.2.2",
   "semver", "0.2.0",
-  "servr", "0.21",
+  "servr", "0.23",
   "sets", "1.0-18",
-  "sfsmisc", "1.1-7",
-  "showtext", "0.9-1",
+  # This will raise an expected warning during image builds, but should still work
```
Comment (GeorgianaElena) [inline deployments/utoronto/image/install.R]: This shouldn't happen anymore
```suggestion
```

**Comment by GeorgianaElena** [comment]
I believe the `sf` package install issue was because of `conda` installing the latest version of `libgdal` and `gdal` as a dependency of ones of the pkgs listed in `environment.yaml`.

Anyway, installing the R packages before the Python ones, solves this situation and it improves the caching of the R pkgs install, which takes forever.

@choldgraf, if [`Build Image`](https://github.com/utoronto-2i2c/jupyterhub-deploy/pull/116/checks?check_run_id=3724634460) is green, do you want to merge this, so that the image gets pushed and deployed to staging, then ask the instructor if everything looks good?

**Comment by choldgraf** [comment]
@GeorgianaElena good call! (and thanks for debugging that install issue, you are a wizard!)

I just went ahead and merged https://github.com/utoronto-2i2c/jupyterhub-deploy/pull/119 and once that is happy (aka I don't have to revert), I will merge this one into staging and we can test it out 👍 

edit: argh it looks like the build is not happy :-/ though the errors look _vaguely_ like they aren't related to this PR? I will try re-running

**Comment by choldgraf** [comment]
wohooo builds are happy, so I am merging!

**Pull request merged**